src/api/nodeRpcCall.js

```diff
@@ -1362,29 +1362,68 @@ const createProposalAndVote = async (threshold, proposalContent, vote) => {
   return [proposal, voteAye];
 };
 
-const congressSendLlm = async ({ walletAddress, transferToAddress, transferAmount }) => {
+const congressProposeSpend = async ({ walletAddress, blockNumber, spendProposal }) => {
   const api = await getApi();
 
   const threshold = await congressMajorityThreshold();
-
-  const executeProposal = api.tx.llm.sendLlm(transferToAddress, transferAmount);
+  const remark = api.tx.llm.remark('Congress spending');
+  const transferAndRemark = api.tx.utility.batchAll([spendProposal, remark]);
+  const dayToBlocks = (24 * 60 * 60) / 6;
+  const motionDuration = 7 * dayToBlocks;
+
+  // CouncilAccount execute forces Congress to use scheduler with at least 4 days delay.
+  // We add 1 extra day in case of any desync between signing and sending tx.
```

Comment:
There's no `desync`, there's nothing that kept in sync.
The additional 1 day is for some person to actually Close (a.k.a. execute) the motion after it ends, which is done manually.